src/widgets/Card/Card.tsx

```diff
@@ -0,0 +1,37 @@
+import { NavLink, useSearchParams } from 'react-router-dom';
+import { Character, CharactersProps } from '../../types';
+import { getCharacterId } from '../../utils/getCharacterId';
+import './Card.css';
+
+const Cards = (props: CharactersProps): JSX.Element => {
+  const [searchParams] = useSearchParams();
+
+  const handleClick = () => {
+    localStorage.setItem('searchUrl', searchParams.toString());
+  };
+
+  return (
+    <ul className="cards__container">
+      {props.characters?.length > 0 ? (
```

Comment:
```suggestion
      {characters?.length > 0 ? (
```you can write shorter